### PR TITLE
Feature/capture premium features

### LINKED ISSUE #1351: Capture the premium features
Use the `PremiumFeature.js` component to capture all the premium features. Remeber just do it for Android devices. 

List of Premium Features:
- Online NeuralVoices
- Copy public boards
- PublishBoards
- CopyTiles
- Paste tiles
- Usage Analytics',
- Share boards
- Ads Free
- Export to open board format
- SharePhrases
- Private support channel
- Voice recording for tiles

### PR DESCRIPTION
**Pull request by tomivm**
close #1351

### CODE REVIEW

**Comment by RodriSanchez1** [comment]
- For share board we can do same than the share button on nav bar. That show the share modal, and when you try to share, the feature blocked modal shows up.
- When a online language is selected, a online voice is seted. This avoid the premium modal and breaks the functionality of capture premium feature. I think we can separate the managment of the online voices in other PR

**Comment by tomivm** [comment]
Good review! 
During the change language from language setting, The app is remounted and the premiumRequiredModal is positioned behind the settings full-screen dialog.

File: src/components/Board/BoardShare/BoardShare.component.js [hunk 1]
```diff
@@ -134,75 +137,77 @@ const BoardShare = ({
         </div>
         {isLogged && (
           <div className="ShareDialog__socialIcons">
-            <Button disabled={!isPublic} onClick={onCopyLink} color="primary">
-              <div className="ShareDialog__socialIcons__copyAction">
-                <div>
-                  <CopyIcon />
+            <PremiumFeature>
```
Comment (RodriSanchez1) [inline src/components/Board/BoardShare/BoardShare.component.js:140]: The styles on labels are changend. Also on share you phrase share

Comment (tomivm) [reply src/components/Board/BoardShare/BoardShare.component.js:140]: this issue is not related to these changes.

**Review by RodriSanchez1 — APPROVED** [review]

**Pull request merged**